package.json

```diff
@@ -105,13 +110,13 @@
     "extract-strings": "./utils/extract-strings.js",
     "flow:ci": "flow check --max-workers 0",
     "lint": "eslint --config .eslintrc.js . --ext .js --ext .jsx",
-    "lint:timing": "TIMING=1 yarn lint",
+    "lint:timing": "cross-env TIMING=1 yarn lint",
     "publish:ci": "node utils/pre-publish-check-ci.js && git diff --stat --exit-code HEAD && yarn build && yarn extract-strings && changeset publish",
     "sloc": "sloc packages --exclude node_modules",
     "test": "yarn jest",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "cypress": "cypress open-ct",
-    "cypress:ci": "cypress run-ct"
+    "cypress:ci": "cross-env COVERAGE=1 cypress run-ct --env COVERAGE=1"
```

Comment:
I wasn't able to replicate the original solution of add a config params to the babel-loader that could be passed to the babel config.  For some reason webpack didn't like the config options so here we are.  The `cross-env` environment variable is used by the babel config while the `--env` one is used by cypress.

Reply:
I wonder if we should name them differently so it's easier to differentiate which one is used where.  Like `BABEL_COVERAGE` and `CYPRESS_COVERAGE`?